src/mds/heartbeat/chunkserver_healthy_checker.cpp

```diff
@@ -134,8 +134,8 @@ void ChunkserverHealthyChecker::UpdateChunkServerOnlineState(
 
 bool ChunkserverHealthyChecker::TrySetChunkServerRetiredIfNeed(
     const HeartbeatInfo &info) {
-    // Disregard when chunkserver isn't in OFFLINE status
-    if (OnlineState::OFFLINE != info.state) {
+    // Disregard when chunkserver is in ONLINE status
+    if (OnlineState::ONLINE == info.state) {
```

Comment:
Why pendding empty chunkserver is unstale ？

Reply:
when you stop this chunkserver, this chunkserver state change from online to unstable.